app/controllers/observations_controller/form_helpers.rb

```diff
@@ -84,20 +84,17 @@ def init_project_vars
     @projects = User.current.projects_member(order: :title,
                                              include: :user_group)
     @project_checks = {}
-    @projects.each do |proj|
-      @project_checks[proj.id] = (proj.open_membership &&
```

Comment:
@mo-nathan I didn't understand the rationale for:
- having any projects checked initially
- treating open projects diferently from closed projects

Reply:
Initially makes sense to me since I think most people most of the time will want the same selections that they just made for the previous observation.  The reason for making it optional was to avoid changing behavior for closed projects.  However, I now think they should be the same so I think this should just be `proj.current?`.  However, I don't think this has to happen in this PR.